### ISSUE TITLE
how to change lip movement with Deepfacelab?

### ISSUE THREAD

**Issue by github-jen** [iperov/DeepFaceLab#887]

**Comment by test1230-lab** [comment]
1) capture yourself with your own speech repeating same head direction as celeb in target video
2) train regular deepfake model with celeb faces from target video as src, and your face as dst
3) merge celeb face onto your face with raw-rgb mode
4) compose masked mouth with target video in AfterEffects

**Comment by rakanshaker** [comment]
Fair enough, but what if I want my face on the celeb's face saying something new, in their video. ex: I want to use famous movie footage to make a music video for my song. Couldn't I train the celeb model like in step 2 & 3, then use that result as the new dst?

**Comment by ghost** [comment]
This is not an issue (or the issue is not related to deepfacelab), please close it.
